client/src/actions/noticesActions.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import produce from 'immer'
+import {get} from 'lodash'
+
+import type {GenericAction, Path} from '../types/reduxTypes'
+import type {State} from '../state'
+
+export const receiveCompanyDetails = (
```

Comment:
there'll be a helper function for this in https://github.com/verejnedigital/verejne.digital/pull/44 - but it's fine like this if this PR get's merged first, and we can unify afterwards